src/metabase/models/collection/graph.clj

```diff
@@ -78,16 +79,23 @@
                                                  [:not [:like :location (h2x/literal (format "/%d/%%" collection-id))]]))}]
     (set (map :id (mdb.query/query honeysql-form)))))
 
+(defn- calculate-perm-groups [collection-namespace group-id->perms collection-ids]
+  (into {}
+        #_:clj-kondo/ignore
```

Comment:
Why ignore this when it's possible to write our own?

1) there's an open PR for it on claypoole
2) there's a repo with the literal kondo hooks code (`io.github.clj-kondo/config-org.clj-commons-claypoole`), that we can include and get the linting hooks setup, but it doesn't have a license, so we can't use it
3) I could copy the hooks over, and then if they change we have to maintain it
4) Ignore it for now

So I went with 4